infrastructure/kubernetes/app-backend-rc.yml

```diff
@@ -28,7 +28,7 @@ spec:
         app: app-backend-rc
     spec:
       containers:
-      - image: "teamdigitale/italia-backend:v1.0.0-rc8"
+      - image: "teamdigitale/italia-backend:v1.0.1"
```

Comment:
Perchè non passiamo alla v1.0.0?

Reply:
questa e' un istanza di test, con cui abbiamo testato la 1.0.0-rc8, che abbiamo trasformato in 1.0.0 e deployato qui: https://github.com/teamdigitale/io-infrastructure/commit/3facf57b8b0b1133b4531c266db358867bec3be8